frontend/pages/organizations/[id]/about.vue

```diff
@@ -67,6 +71,7 @@ definePageMeta({
 });
 
 import { Organization } from "~~/types/organization";
+import IconOrganizationStatus from "../../../components/icon/IconOrganizationStatus.vue";
```

Comment:
@to-sta, I think that this import here is unnecessary :) Nuxt covers all the component imports for us automatically, and we just need to load in the types 😊

Reply:
@andrewtavis thanks for the info. Will keep that in mind 👍